testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -160,7 +160,7 @@ class MiniAppDisplayActivity : BaseActivity() {
         miniAppMessageBridge.setAdMobDisplayer(AdMobDisplayer(this@MiniAppDisplayActivity))
         miniAppMessageBridge.allowScreenOrientation(true)
 
-        val userInfoBridgeDispatcher = object : UserInfoBridgeDispatcher() {
```

Comment:
We need to update the userguide as well according to this change.

Reply:
updated.